release/scan_docker_images.sh

```diff
@@ -2,6 +2,39 @@
 
 source ../_liferay_common.sh
 
+function check_usage {
+	if [ -z "${LIFERAY_IMAGE_NAMES}" ]
+	then
+		print_help
+	fi
+
+	lc_cd "$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")"
+
+	_RELEASE_ROOT_DIR="${PWD}"
+
+	LIFERAY_COMMON_LOG_DIR="${_RELEASE_ROOT_DIR}/logs"
+
+	mkdir -p "${LIFERAY_COMMON_LOG_DIR}"
+}
+
+function main {
+	check_usage
+
+	lc_time_run scan_docker_images
+}
+
+function print_help {
+	echo "Usage: LIFERAY_IMAGE_NAMES=<image name> ${0}"
+	echo ""
+	echo "The script reads the following environment variables:"
+	echo ""
+	echo "    LIFERAY_IMAGE_NAMES: DXP or portal version of Docker image"
+	echo ""
+	echo "Example: LIFERAY_IMAGE_NAMES=liferay/dxp:2025.q1.5-lts ${0}"
```

Comment:
@lucasmiranda0, can you please make sure people are aware that this variable supports more than one Docker image?